kerastuner/engine/execution.py

```diff
@@ -1,103 +0,0 @@
-# Copyright 2019 The Keras Tuner Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-"""Execution class."""
-
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
-import time
-import os
-import json
-
-from . import metrics_tracking
-from ..abstractions.tensorflow import TENSORFLOW_UTILS as tf_utils
-
-
-class Execution(object):
```

Comment:
Since we should keep the mechanism of metric denoising via multiple training runs (i.e. execution), we should preserve the Execution class as an object storing per-execution information such as checkpointing directory and per-execution metrics.

Reply:
Discussed offline, created a MultiExecutionTuner that handles multiple executions